tests/verify_collection.py

```diff
@@ -267,8 +280,8 @@ def get_lat_lon_var_names(dataset: xarray.Dataset, collection_variable_list: Lis
         logging.warning("Unable to find lat/lon vars using 'units' attribute")
 
     # Out of options, fail the test because we couldn't determine lat/lon variables
-    pytest.fail(f"Unable to find latitude and longitude variables.")
-
+    logging.warning(f"Unable to find latitude and longitude variables in this group.")
```

Comment:
Can you add the group name as an optional parameter to the function so we can log the name of the group here to help with debugging?